<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_Sf2</class>
 <widget class="QWidget" name="Page_Sf2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="1">
         <widget class="QCheckBox" name="check_24bits">
          <property name="text">
           <string>24 bits</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Auteur</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Copyright</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_author">
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_copyright">
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Commentaires</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="TextEdit" name="textEdit_Com"/>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_date">
            <property name="maxLength">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_setNow">
            <property name="text">
             <string>Date courante</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Produit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_product">
            <property name="maxLength">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Échantillons</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Fichier</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_filename">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Version Soundfont</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_nbInst">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_nbInstGen">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Nombre d'instruments</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Nombre d'échantillons</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Nombre de paramètres (instruments)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_nbPrst">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Nombre de paramètres (presets)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_nbSmpl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Nombre de presets</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_nbPrstGen">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_sfVersion">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nom et version ROM</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_romVersion">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Sound engine</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_soundEngine">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Logiciel(s) d'édition</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_software">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>page_sf2.h</header>
   <slots>
    <signal>editingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_author</tabstop>
  <tabstop>lineEdit_copyright</tabstop>
  <tabstop>lineEdit_date</tabstop>
  <tabstop>pushButton_setNow</tabstop>
  <tabstop>lineEdit_product</tabstop>
  <tabstop>textEdit_Com</tabstop>
  <tabstop>check_24bits</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_name</sender>
   <signal>editingFinished()</signal>
   <receiver>Page_Sf2</receiver>
   <slot>setName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit_Com</sender>
   <signal>editingFinished()</signal>
   <receiver>Page_Sf2</receiver>
   <slot>setCommentaire()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_product</sender>
   <signal>editingFinished()</signal>
   <receiver>Page_Sf2</receiver>
   <slot>setProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_date</sender>
   <signal>editingFinished()</signal>
   <receiver>Page_Sf2</receiver>
   <slot>setDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_author</sender>
   <signal>editingFinished()</signal>
   <receiver>Page_Sf2</receiver>
   <slot>setAuthor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_setNow</sender>
   <signal>clicked()</signal>
   <receiver>Page_Sf2</receiver>
   <slot>setNow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_copyright</sender>
   <signal>editingFinished()</signal>
   <receiver>Page_Sf2</receiver>
   <slot>setCopyright()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_24bits</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Page_Sf2</receiver>
   <slot>set24bits(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set24bits(int)</slot>
  <slot>setName()</slot>
  <slot>setCopyright()</slot>
  <slot>setAuthor()</slot>
  <slot>setDate()</slot>
  <slot>setProduct()</slot>
  <slot>setCommentaire()</slot>
  <slot>setNow()</slot>
 </slots>
</ui>
